packages/react/src/progress-indicator/ProgressIndicator.stories.tsx

```diff
@@ -12,38 +16,54 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 const exampleLinkItems: ProgressIndicatorItem[] = [
-	{ label: 'Introduction', status: 'done', href: '#' },
-	{ label: 'Submit evidence', status: 'saved', href: '#' },
-	{ label: 'Organisations', status: 'started', href: '#' },
-	{ label: 'Business contacts', status: 'error', href: '#' },
-	{ label: 'Case studies', status: 'todo', href: '#' },
-	{ label: 'Attachments', status: 'started', href: '#', isActive: true },
-	{ label: 'Review and submit', status: 'blocked', href: '#' },
+	{ label: 'Introduction', status: 'done', href: '/introduction' },
+	{ label: 'Submit evidence', status: 'saved', href: '/evidence' },
+	{ label: 'Organisations', status: 'started', href: '/organisations' },
+	{ label: 'Business contacts', status: 'error', href: '/contacts' },
+	{ label: 'Case studies', status: 'todo', href: '/case-studies' },
+	{ label: 'Attachments', status: 'started', href: '/attachments' },
+	{ label: 'Review and submit', status: 'blocked', href: '/review' },
 ];
 
+const exampleLevelTwoLinkItems: ProgressIndicatorItemWithLevelTwoItems[] =
+	exampleLinkItems.map((item) =>
+		item.label === 'Organisations'
+			? {
+					...item,
+					items: [
+						{
+							label: 'Change organisation name',
+							href: '/organisations/change-name',
+						},
+					],
+			  }
+			: item
+	);
+
 const exampleButtonItems: ProgressIndicatorItem[] = [
-	{ label: 'Introduction', status: 'done', onClick: console.log },
-	{ label: 'Submit evidence', status: 'saved', onClick: console.log },
-	{ label: 'Organisations', status: 'started', onClick: console.log },
-	{ label: 'Business contacts', status: 'error', onClick: console.log },
-	{ label: 'Case studies', status: 'todo', onClick: console.log },
+	{ label: 'Introduction', status: 'done', onClick: console.info },
```

Comment:
Why use `info` instead of `log`? But do we even want to showcase this now?

Reply:
Yeah, I'll remove the buttons example. I actually forgot to remove this one. All the other docs references to buttons are gone.

>Why use info instead of log?

It helps to distinguish between accidentally left in debugging and deliberate message printing in the console. Just about everyone uses `console.log` for debugging. `console.info` is effectively the same, AFAICT, so I just change them to make it easier to spot. Happy to revert, if it's an issue.

Reply:
No, that reasoning sounds fine with me. Never seen it used before, but should be fine. I wonder if other devs might be confused though (as I initially was)?

Reply:
Hmm yeah, I should probably just raise a ticket to do it across the repo and add the reasoning to the README rather than just change as I go.

Reply:
- [ ] Remove button story